src/stateAggregator/globalInfoConfig.ts

```diff
@@ -1,123 +0,0 @@
-/*
- * Copyright (c) 2020, salesforce.com, inc.
- * All rights reserved.
- * Licensed under the BSD 3-Clause license.
- * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-import { AnyJson, isPlainObject, JsonMap, Optional } from '@salesforce/ts-types';
-import { Global } from '../global';
-import { ConfigFile } from '../config/configFile';
-import { ConfigValue } from '../config/configStore';
-import { SfdxDataHandler } from './sfdxDataHandler';
-import { GlobalInfoOrgAccessor } from './accessors/orgAccessor';
-import { GlobaInfoTokenAccessor } from './accessors/tokenAccessor';
-import { GlobalInfoAliasAccessor } from './accessors/aliasAccessor';
-import { SfInfo, SfInfoKeys } from './types';
-import { GlobalInfoSandboxAccessor } from './accessors/sandboxAccessor';
-
-export function deepCopy<T extends AnyJson>(data: T): T {
-  return JSON.parse(JSON.stringify(data)) as T;
-}
-
-/**
- * @deprecated use StateAggregator instead.
- */
-export class GlobalInfo extends ConfigFile<ConfigFile.Options, SfInfo> {
```

Comment:
there's some of this used by pdt
https://github.com/forcedotcom/salesforcedx-apex/search?q=globalinfo